### DISCUSSION TITLE
笔记如何直接关联某特定笔记,以及复制行与复制节有何区别?

### DISCUSSION THREAD

**Discussion by shikily** [windingwind/zotero-better-notes#1178] [Q&A] [answered]
目前做好笔记后,想要关联其他文献,貌似只能:1)插入引注,只能是整篇文献;2)“相关”处直接添加,似乎也是指向整篇文献;3)选中笔记里某段文字“添加链接“,才能链接到某个具体笔记.
想请教是否有更便捷的方法可以让笔记和笔记之间相互链接?用3)的方法有个不方便的点在于,有时想整个a笔记指向b笔记,而不是其中具体某个表述,这样虽然还是可以选中a笔记中某文字然后加b笔记的链接,但会觉得有点别扭.
以及笔记右上角的选项里,“复制行(L0)”与“复制节(Sec.)“有何区别?
🙏

**Comment by windingwind** [comment, marked as answer]
笔记编辑界面左上角 链接精灵，可以创建从当前笔记的入链和出链

复制行=复制当前光标行的链接。
复制节=复制当前光标所在标题（章节）的链接。

**Reply by shikily** [reply]
感谢回复!

**Comment by CodexDive** [comment]
能否为复制节指定快捷键方便复制，通过链接精灵挺低效的。

**Reply by windingwind** [reply]
你可以比较一下使用链接精灵和手动复制再粘贴的操作步数。即使有快捷键也明显是链接精灵操作简单

另外功能请求请开issue
